system/lang/pl.json

```diff
@@ -10,26 +10,48 @@
     "CustomTruth": "Własna Prawda",
     "Roll": "Rzuć",
     "Chat": "Czat",
-    "Rolling": "Losuje..",
+    "SendToChat": "Wyślij tekst na czat: <i>{move}</i>",
+    "CopyTextToClipboard": "Kopiowanie tekstu do schowka",
+   "Rolling": "Losuje..",
     "Bonus": "Bonus",
+    "Adds": "Dodaje",
     "Versus": "vs.",
-    "CappedAt10": "Capped at 10",
+    "CappedAt10": "Ograniczenie do 10",
     "Strong_hit": "Sukces!",
     "Strong_hit_match": "Triumf! (match)",
     "On_a_strong_hit": "Przy Pełnym Sukcesie",
     "Weak_hit": "Częściowy Sukces",
     "On_a_weak_hit": "Przy Częściowym Sukcesie",
-    "Miss": "Porazka",
+    "Miss": "Porażka",
     "Miss_match": "Krytyczna Porazka(match)",
     "On_a_miss": "Przy miss",
```

Comment:
would localizing 'miss' be more appropriate here?

Reply:
If you mean that translation would be more appropriate "Porażka" (which is opposite word to success) is the best polish word for describing english "miss" in this context (also polish/translated pbta/fitd games are using this word for it :))

Reply:
I mean this line:
```
"On_a_miss": "Przy miss",
```

Reply:
Oh goddess. Where are my eyese? Yes, you are correct, again thank you i'll change it asap